### PR TITLE
Capstone-project

### PR DESCRIPTION
**Pull request by Ambrosegithub**
- [ ] add  2 index file for both home and about page.
- [ ] add 2 css file for both pages.
- [ ] add javascript file.
- [ ] create speaker section dynamically with Javascript.
- [ ] add Readme file for project description.
- [ ] add images.
- [ ] use HTML CSS AND JAVASCRIPT FOR THE PROJECT.
- [ ] use linter and git hub flow.

please kindly help out if you are the reviewer reviewing this code, I have exhausted all my knowledge about linter and I seem not to get it solved. 
I would really appreciate the assistance rendered thanks.

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @Ambrosegithub,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] Some linters checks are not passed. So please fix that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by Ismailco** [comment]
@Ambrosegithub, kindly fix your linter errors so you can get a review ASAP

**Comment by Kingstalux** [comment]
Hi @Ambrosegithub , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ 

- [ ] You have submitted a project, that has failing linters checks. Kindly fix those before submitting for a review 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1,2 +1,101 @@
-# Capstone-project1
-[Capstone project explained.](https://www.loom.com/share/f995084a14bb430891f5c21e476bd021).
\ No newline at end of file
+# HTML & CSS3 & JavaScript Course
```
Comment (arslanbisharat) [inline README.md]: - [ ] Thank you so much for adding this readme file but this is not requested by Microverse. Please add a proper readme. You can check the [templates](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,101 @@
...
+
+If you are not familiar with linters, read [root level README](../README.md).
```
Comment (arslanbisharat) [inline README.md]: - [ ] The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,101 @@
...
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+4. Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
+5. Fix linter errors.
+6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/cli#autofixing-errors) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+### [ESLint](https://eslint.org/)
+
+1. Run 
+   ```
```
Comment (arslanbisharat) [inline README.md]: - [ ] Please add a link to the live version of your project so it is easily accessible for anyone

File: README.md [hunk 4]
```diff
@@ -1,2 +1,101 @@
...
+   npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+   ``` 
+   *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
+
```
Comment (arslanbisharat) [inline README.md]: - [ ] Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include

-     A description of the project.
-     A demo of the project features (different pages and different screen sizes).
-     You should also highlight some interesting piece of code or something you built that you are very proud of.
-

File: index.html [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+    <title>Capstone Project</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/d6cd3c7604.js"
+      crossorigin="anonymous"
+    ></script>
```
Comment (arslanbisharat) [inline index.html:16]: - [ ] The bottom header is not fixed and doesn't reflect the design when scrolling up or down. 
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+            <img src="./images/partner1.png" alt="" />
+          </div>
+          <div class="partners-logo">
+            <img src="./images/WTO.png" alt="" />
+            <h4>WTO</h4>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer>
```
Comment (arslanbisharat) [inline index.html:193]: - [ ] On the desktop the partner section and footer doesn't match with the given design. Please make sure you follow the design. 

image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+                  >YEHO campaign</a
+                >
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <main class="Home-page" id="Home-page">
+      <section class="home-cont">
```
Comment (arslanbisharat) [inline index.html:84]: - [ ] Please make sure you follow the given layout and color scheme. 
image

File: About.html [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+                            id="img-header"
+                            src="./images/IMG-header-removebg-preview.png"
+                            alt=""
+                          />
+                        </a>
+                      </div>
+                      <div class="nav-link-container">
+                <ul class="nav-links">
+                    <li>
+                        <a id="about" href="./index.html" class="link">About</a>
```
Comment (arslanbisharat) [inline About.html]: - [ ] This is index page link so please remove About from here and add Home here,

File: About.html [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+        <p>Hello Sharing World!</p>
+        <h1>WORLD <br>ENVIRONMENTAL<br>HEALTH DAY 2022</h1>
+        </div>
+        <div class="pages">
+        <P> &nbsp; &nbsp; &nbsp; &nbsp; The World Health Environmental Day <br> &nbsp; is an annual  event to commemerate the sucesses <br> ably attain and 
+            promote health for all (SDGs) goals. <br> this year, the YEHO FORUM has taken it upon itself to <br> join others professionals from the international allies <br> 
+            in making this august event a huge success.  Inline with this,<br> the YEHO FORUM also deem it neccessary to carry <br> awareness campain in Schools, public parks, remote <br> &nbsp;
+           &nbsp; areas and so on. On preventive measures and <br> &nbsp; &nbsp; &nbsp; &nbsp; healthy living. 
+           &nbsp; &nbsp;&nbsp; &nbsp;&nbsp; in quest of attaining the <br>&nbsp; &nbsp;&nbsp; &nbsp;&nbsp; &nbsp;Sustainable Development Goals(SDGs).
+        </P>
```
Comment (arslanbisharat) [inline About.html:91]: - [ ] On mobile screen the menu button color and layout should match the given in the design. 
image

File: About.html [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+            </div>
+            </div>
+        </nav>
+    </header>
+    <main class="Home-page">
+        <section class="home-cont">
+        <div class="home-text">
+        <p>Hello Sharing World!</p>
+        <h1>WORLD <br>ENVIRONMENTAL<br>HEALTH DAY 2022</h1>
+        </div>
```
Comment (arslanbisharat) [inline About.html:84]: - [ ] On mobile screen there are multiple issues present. For example, the text is not aligned. 

image

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Ambrosegithub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: About.html (hunk 9 above)
Comment (Ambrosegithub) [reply About.html:91]: DONE

**Comment by Ambrosegithub** [comment]
DONE

**Comment by Ambrosegithub** [comment]
DONE

File: About.html (hunk 9 above)
Comment (Ambrosegithub) [reply About.html:91]: DONE

File: About.html (hunk 8 above)
Comment (Ambrosegithub) [reply About.html]: DONE

Comment (Ambrosegithub) [reply About.html]: DONE

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @Ambrosegithub     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Suggestion

- [ ] [OPTIONAL] Please add the image on top left of the speakers images as in the design

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**